docs/tutorials/multi-d-datasets.ipynb

```diff
@@ -358,7 +408,18 @@
     "When using `numpy` to compute the mean:\n",
```

Comment:
Just a little further up (could not comment on the right line):
" 1. Use `ds.mean` to compute the mean of the data for Alice along the Z dimension.\n"
should probably be
" 1. Use `mean` to compute the mean of the data for Alice along the Z dimension.\n"